eventtracking/backends/logger.py

```diff
@@ -45,7 +45,7 @@ def send(self, event):
 class DateTimeJSONEncoder(json.JSONEncoder):
     """JSON encoder aware of datetime.datetime and datetime.date objects"""
 
-    def default(self, obj):  # pylint: disable=method-hidden
+    def default(self, obj):  # lint-amnesty, pylint: disable=arguments-differ, method-hidden
```

Comment:
Ah - it was unclear whether you added them manually or automatically, since they were in the same commit as all other changes. Carry on...